test/cli/repo.js

```diff
@@ -18,4 +19,30 @@ describe('repo', () => runOnAndOff((thing) => {
       expect(out).to.eql(`${repoVersion}\n`)
     })
   })
+
+  // Note: There are more comprehensive GC tests in interface-js-ipfs-core
```

Comment:
Can we also have a test (or two) that mocks IPFS and asserts that `stream-errors` is passed from argv to the mock correctly and that the output is formatted appropriately given canned results from the mock?

Ideally I'd like to move towards having just a few e2e CLI tests (since we have comprehensive core tests) and many unit tests for the handlers that simply ensure args are converted appropriately for core and that output from core is formatted appropriately in the CLI.

I think it would also help to explicitly group types of tests in a directory "unit", "e2e" etc.

refs https://github.com/ipfs/js-ipfs/issues/2131
cc @hugomrdias

Reply:
yes please only test cli logic in cli tests, you can check the ipns cli tests for examples 

there are still some weird in there because of how the cli works, but we will make that better this quarter